experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tableExampleSchemaConverter.spec.ts

```diff
@@ -118,21 +120,29 @@ describe("LlsSchemaConverter", () => {
 		for (const typeName of typeNames) {
 			const nodeSchema = fullSchemaData.nodeSchema.get(typeName);
 			assert(nodeSchema !== undefined);
-			nodeSchema.objectNodeFields.forEach((field, fieldKey) => {
-				if (field.types) {
-					field.types.forEach((type) => {
-						assert(
-							typeNames.has(type),
-							`Missing type "${type}" in tree schema "${typeName}" for a local field "${fieldKey}"`,
-						);
-					});
-				}
-			});
-			if (nodeSchema.mapFields?.types) {
+			if (nodeSchema instanceof ObjectNodeSchema) {
+				nodeSchema.objectNodeFields.forEach((field, fieldKey) => {
+					if (field.types) {
+						field.types.forEach((type) => {
+							assert(
+								typeNames.has(type),
+								`Missing type "${type}" in tree schema "${typeName}" for a local field "${fieldKey}"`,
+							);
+						});
+					}
+				});
+			} else if (nodeSchema instanceof MapNodeSchema && nodeSchema.mapFields.types) {
 				nodeSchema.mapFields.types.forEach((type) => {
 					assert(
 						typeNames.has(type),
-						`Missing type "${type}" in tree schema "${typeName}" for extra local fields`,
+						`Missing type "${type}" in tree schema "${typeName}" for map fields`,
+					);
+				});
+			} else if (nodeSchema instanceof FieldNodeSchema && nodeSchema.info.types) {
```

Comment:
Nit: might be worth adding a quick note that this handles the list case, since I don't think it's intuitively obvious that lists fall under the field node category.

Reply:
"list" (being renamed to ArrayNode) is not a thing at this abstraction level (this package is written using the flex tree schema APIs). This package explicitly uses fields nodes.